variables.tf

```
@@ -34,3 +52,9 @@ variable "node_security_group_id" {
   description = "Security Group ID for EKS nodes"
   type        = string
 }
+
+variable "volume_size" {
+  description = "EBS volume size for Karpenter nodes in GiB"
+  type        = string
+  default     = "60Gi"
```

With the additional mitigations you have put in place, I would like to think this number can come down:

```suggestion
  default     = "40Gi"
```

This will reduce the costs. This is still _double_ the default.